.gitignore

```diff
@@ -8,6 +8,9 @@ prof/
 .venv/
 .mypy_cache/
 __pycache__/
+.idea/
+.vscode/
```

Comment:
For folders specific to your own environment, please use **.git/info/exclude** instead.